### PR TITLE
feat(bridge-ui): Tabs Component

### PR DESCRIPTION
**Pull request by jscriptcoder**
New Tabs component, reusable and fully accessible

### CODE REVIEW

**Review by dantaik — DISMISSED** [review]

File: packages/bridge-ui/src/pages/home/Home.svelte [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
   let bridgeWidth: number;
   let bridgeHeight: number;
 
   $: activeTab = $location.replace('/', '').startsWith('transactions')
     ? 'transactions'
     : 'bridge';
+
+  // TODO: do we really need all these tricks to style containers
+  //       Rethink this part: fluid, fixing on small screens
```
Comment (jscriptcoder) [inline packages/bridge-ui/src/pages/home/Home.svelte:18]: Typo here, I meant, fixing on bigger screens

File: packages/bridge-ui/src/pages/home/Home.svelte [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
   import { transactions } from '../../store/transactions';
   import BridgeForm from '../../components/form/BridgeForm.svelte';
   import TaikoBanner from '../../components/TaikoBanner.svelte';
   import Transactions from '../../components/Transactions.svelte';
+  import { Tabs, TabList, Tab, TabPanel } from '../../components/Tabs';
 
   let bridgeWidth: number;
   let bridgeHeight: number;
 
   $: activeTab = $location.replace('/', '').startsWith('transactions')
```
Comment (cyberhorsey) [inline packages/bridge-ui/src/pages/home/Home.svelte:13]: this seems brittle, if we change the route name, or if we add another API route like `/transactionsForChain` and only show one chain's transactions, or something.

File: packages/bridge-ui/src/components/Tabs/Tab.svelte [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { link } from 'svelte-spa-router';
+  import { getContext } from 'svelte';
+  import type { Writable } from 'svelte/store';
+
+  export let href: string = '';
+  export let name: string = '';
+
+  const activeTab = getContext<Writable<string>>('activeTab');
```
[the remark this answers is not in the document]
Comment (cyberhorsey) [reply packages/bridge-ui/src/components/Tabs/Tab.svelte:9]: this is actually a good use case for a `Symbol`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol

File: packages/bridge-ui/src/pages/home/Home.svelte (hunk 2 above)
Comment (jscriptcoder) [reply packages/bridge-ui/src/pages/home/Home.svelte:13]: Agree. Let me change that

File: packages/bridge-ui/src/components/Tabs/Tab.svelte (hunk 3 above)
[the remark this answers is not in the document]
Comment (jscriptcoder) [reply packages/bridge-ui/src/components/Tabs/Tab.svelte:9]: That's actually a great idea, but I'd like to keep the encapsulation so I'm gonna expose the key to be accessible across the children:

```
<-- Tabs.svelte -->
<script lang="ts" context="module">
  export const key = Symbol();
</script>
```

**Review by cyberhorsey — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**